ext/date/php_date.c

```diff
@@ -4666,6 +4667,28 @@ PHP_METHOD(DatePeriod, getDateInterval)
 }
 /* }}} */
 
+/* {{{ proto DatePeriod::getRecurrences()
+   Get recurrences.
+*/
+PHP_METHOD(DatePeriod, getRecurrences)
+{
+        php_period_obj   *dpobj;
+        php_date_obj     *dateobj;
+
+        if (zend_parse_parameters_none() == FAILURE) {
+                return;
+        }
+
+        dpobj = Z_PHPPERIOD_P(ZEND_THIS);
+
+        if (!dpobj->recurrences) {
```

Comment:
@derickr seems `dpobj->recurrences` by default equals `1` instead of `NULL` maybe we should substract `1` to the value and return if the substracted value equals `0` :thinking: